data/presets/amenity/doctors/gynaecology.json

```
@@ -28,5 +30,5 @@
         "key": "healthcare:speciality",
         "value": "gynaecology"
     },
-    "name": "Gynecologist"
+    "name": "Obstetrician/Gynecologist"
```

Let's also add the individual terms as `aliases`

```suggestion
    "name": "Obstetrician/Gynecologist",
    "aliases": [
        "Gynecologist",
        "Obstetrician"
    ]
```